dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileSetReader.cpp

```diff
@@ -185,16 +185,12 @@ size_t ColumnFileSetReader::readRows(
             }
         }
     }
+    auto lac_bytes_collector = std::make_unique<LACBytesCollector>(context.scan_context->resource_group_name);
```

Comment:
How about being member variable of `ColumnFileSetReader`?

Reply:
done